### PR TITLE
Updated examples in Relay spec

### PR DESCRIPTION
**Pull request by v-Ajnava**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [X] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

### CODE REVIEW

**Review by veronicagg — CHANGES_REQUESTED** [review]
Please review errors at https://travis-ci.org/Azure/azure-rest-api-specs/jobs/214456216#L1054

Also, I see that there are some models being renames and a property being removed, are those intentional "breaking changes"?

**Comment by v-Ajnava** [comment]
@veronicagg, can you help on getting the solution for error : https://travis-ci.org/Azure/azure-rest-api-specs/jobs/214518899#L1055. Validation passed locally. not sure why this error not surfaced locally. 

Regarding the renaming Models, After generating code through Autorest, Namespace was renamed as NamespaceModel, as Namespace is reserved keyword and conflicting. So Renamed it to RelayNamespace.

Property removed: 'enabled' was added by mistake.

**Comment by veronicagg** [comment]
Thanks @v-Ajnava . I've tagged this PR as "potential-sdk-breaking-change" due to the renames, because as you're saying namespace name is changing, it appears that this spec is new, so if no SDKs have been generated and shipped out of it yet, then this will not cause a breaking change.

Regarding the semantic, there seems to be a syntax error at https://github.com/v-Ajnava/azure-rest-api-specs/blob/828e6cdf17319e912dd7a0942f377cc9cbb917d3/arm-relay/2016-07-01/examples/NameSpaces/RelayNameSpaceAuthorizationRuleListKey.json#L7 (please remove the last comma)
Have you looked over the linter warning and errors:
- https://travis-ci.org/Azure/azure-rest-api-specs/jobs/214518898#L1079
- https://travis-ci.org/Azure/azure-rest-api-specs/jobs/214518898#L1101 - are the resources flagged actually tracked resources? If they are not, then please ignore the errors, if they are, should location property be marked as required?

**Comment by veronicagg** [comment]
@v-Ajnava additionally, please be aware you'll need to sign a CLA, as indicated above.

**Comment by v-Ajnava** [comment]
@veronicagg, the spec is new and SDK is not shipped. Regarding the flagged resources, those are not tracked resources. I have fixed the other validation errors.

File: arm-relay/2016-07-01/examples/HybridConnection/RelayHybridConnectionAutorizationRuleDelete.json [hunk 1]
```diff
@@ -8,8 +8,7 @@
     "subscriptionId": "8c246e6a-cbc6-4a87-aff1-d6a886f376ef"
   },
   "responses": {
-    "200": {
-      "body": {}
-    }
+    "200": {},
+	"204": {}
```
Comment (veronicagg) [inline arm-relay/2016-07-01/examples/HybridConnection/RelayHybridConnectionAutorizationRuleDelete.json]: minor - fix indenting

File: arm-relay/2016-07-01/examples/HybridConnection/RelayHybridconnectionDelete.json [hunk 2]
```diff
@@ -7,8 +7,7 @@
     "subscriptionId": "8c246e6a-cbc6-4a87-aff1-d6a886f376ef"
   },
   "responses": {
-    "200": {
-      "body": {}
-    }
+    "200": {},
+	"204": {}
```
Comment (veronicagg) [inline arm-relay/2016-07-01/examples/HybridConnection/RelayHybridconnectionDelete.json]: minor - fix indenting

File: arm-relay/2016-07-01/examples/NameSpaces/RelayNameSpaceAutorizationRuleDelete.json [hunk 3]
```diff
@@ -7,8 +7,7 @@
     "subscriptionId": "8c246e6a-cbc6-4a87-aff1-d6a886f376ef"
   },
   "responses": {
-    "200": {
-      "body": {}
-    }
+    "200": {},
+	"204": {}
```
Comment (veronicagg) [inline arm-relay/2016-07-01/examples/NameSpaces/RelayNameSpaceAutorizationRuleDelete.json]: minor - fix indenting

File: arm-relay/2016-07-01/examples/NameSpaces/RelayNameSpaceDelete.json [hunk 4]
```diff
@@ -6,8 +6,8 @@
     "subscriptionId": "8c246e6a-cbc6-4a87-aff1-d6a886f376ef"
   },
   "responses": {
-    "200": {
-      "body": {}
-    }
+    "200": {},
+	"202": {},
```
Comment (veronicagg) [inline arm-relay/2016-07-01/examples/NameSpaces/RelayNameSpaceDelete.json]: minor - fix indenting

File: arm-relay/2016-07-01/examples/Relay/RelayAutorizationRuleDelete.json [hunk 5]
```diff
@@ -8,8 +8,7 @@
     "subscriptionId": "8c246e6a-cbc6-4a87-aff1-d6a886f376ef"
   },
   "responses": {
-    "200": {
-      "body": {}
-    }
+    "200": {},
+	"204": {}
```
Comment (veronicagg) [inline arm-relay/2016-07-01/examples/Relay/RelayAutorizationRuleDelete.json]: minor - fix indenting

File: arm-relay/2016-07-01/swagger/relay.json [hunk 6]
```diff
@@ -1792,6 +1792,7 @@
       "properties": {
         "provisioningState": {
           "type": "string",
+		  "readOnly": true,
```
Comment (veronicagg) [inline arm-relay/2016-07-01/swagger/relay.json]: minor - fix indenting

File: arm-relay/2016-07-01/swagger/relay.json [hunk 7]
```diff
@@ -1951,7 +1945,7 @@
           ],
           "x-ms-enum": {
             "name": "policyKey",
-            "modelAsString": true
+            "modelAsString": false
```
Comment (veronicagg) [inline arm-relay/2016-07-01/swagger/relay.json]: are you intentionally changing this property to a enum that can only be the values you indicated above?https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum

**Comment by veronicagg** [comment]
Changes look good. 
I pointed out a few formatting changes above and you may want to add a description for this property: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/215655165#L1171.
thanks!

**Review by veronicagg — CHANGES_REQUESTED** [review]

File: arm-relay/2016-07-01/examples/HybridConnection/RelayHybridConnectionAutorizationRuleDelete.json (hunk 1 above)
Comment (v-Ajnava) [inline arm-relay/2016-07-01/examples/HybridConnection/RelayHybridConnectionAutorizationRuleDelete.json]: Resolved

File: arm-relay/2016-07-01/examples/HybridConnection/RelayHybridconnectionDelete.json (hunk 2 above)
Comment (v-Ajnava) [inline arm-relay/2016-07-01/examples/HybridConnection/RelayHybridconnectionDelete.json]: Resolved

File: arm-relay/2016-07-01/examples/NameSpaces/RelayNameSpaceAutorizationRuleDelete.json (hunk 3 above)
Comment (v-Ajnava) [inline arm-relay/2016-07-01/examples/NameSpaces/RelayNameSpaceAutorizationRuleDelete.json]: Resolved

File: arm-relay/2016-07-01/examples/NameSpaces/RelayNameSpaceDelete.json (hunk 4 above)
Comment (v-Ajnava) [inline arm-relay/2016-07-01/examples/NameSpaces/RelayNameSpaceDelete.json]: Resolved

File: arm-relay/2016-07-01/examples/Relay/RelayAutorizationRuleDelete.json (hunk 5 above)
Comment (v-Ajnava) [inline arm-relay/2016-07-01/examples/Relay/RelayAutorizationRuleDelete.json]: Resolved

File: arm-relay/2016-07-01/swagger/relay.json (hunk 6 above)
Comment (v-Ajnava) [inline arm-relay/2016-07-01/swagger/relay.json]: Resolved

File: arm-relay/2016-07-01/swagger/relay.json (hunk 7 above)
Comment (v-Ajnava) [inline arm-relay/2016-07-01/swagger/relay.json]: changed to True.

**Comment by v-Ajnava** [comment]
@veronicagg, indention, description and recommended changes are fixed.

File: arm-relay/2016-07-01/examples/HybridConnection/RelayHybridConnectionAutorizationRuleDelete.json (hunk 1 above)
Comment (veronicagg) [inline arm-relay/2016-07-01/examples/HybridConnection/RelayHybridConnectionAutorizationRuleDelete.json]: I still see the same indentation issue.

**Review by veronicagg — CHANGES_REQUESTED** [review]

**Review by veronicagg — APPROVED** [review]

**Pull request merged**